flash/vision/video/classification/data.py

```diff
@@ -0,0 +1,170 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pathlib
+from typing import Any, Callable, Dict, Iterable, List, Optional, Sequence, Tuple, Type, Union
+
+import torch
+from pytorch_lightning.utilities.exceptions import MisconfigurationException
+from torch import nn
+from torch.nn import Module
+from torch.utils.data import Dataset, RandomSampler, Sampler
+from torch.utils.data._utils.collate import default_collate
+from torch.utils.data.dataset import IterableDataset
+from torchvision.datasets.folder import has_file_allowed_extension, IMG_EXTENSIONS, make_dataset
+
+from flash.data.auto_dataset import AutoDataset
+from flash.data.data_module import DataModule
+from flash.data.data_pipeline import DataPipeline
+from flash.data.process import Preprocess
+from flash.utils.imports import _KORNIA_AVAILABLE, _PYTORCH_VIDEO_AVAILABLE
+
+if _KORNIA_AVAILABLE:
+    import kornia.augmentation as K
+    import kornia.geometry.transform as T
+else:
+    from torchvision import transforms as T
+
```

Comment:
augmentations in video need to be handled different since there's an extra dimension. We have the `VideoSequential` obejct that works as follows: https://kornia.readthedocs.io/en/latest/augmentation.container.html#kornia.augmentation.container.VideoSequential

```python

aug_list = VideoSequential(
...     kornia.augmentation.ColorJitter(0.1, 0.1, 0.1, 0.1, p=1.0),
...     kornia.augmentation.RandomAffine(360, p=1.0),
... data_format="BCTHW",
... same_on_frame=True)
```

Reply:
Apart from that, PTV I belive they have their own api based dictionary keys through `ApplyTransformToKey` /cc @tullie

Reply:
Sounds great ! I will use Kornia defaults ones !